### PR TITLE
feat(onboarding): add staging support to the chart

### PR DESCRIPTION
**Pull request by bfabio**

### CODE REVIEW

File: onboarding/values-staging.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    onboarding_port: "3000"
+    onboarding_environment: "staging"
+    onboarding_app_base_url: "https://onboarding-staging.zland.org"
+    onboarding_es_url: "https://elasticsearch.developers.italia.it/indicepa_pec/_search"
+    onboarding_smtp_port: "587"
+    onboarding_smtp_secure: "false"
+    onboarding_email_from: "Team per la Trasformazione Digitale <contatti@developers.italia.it>"
+    onboarding_email_cc: ""
+    onboarding_email_bcc: "contatti@developers.italia.it"
+    onboarding_email_override_recipient_addr: "fabio.bonelli@teamdigitale.governo.it"
```
Comment (bfabio) [inline onboarding/values-staging.yaml:26]: We need a shared email address here

File: onboarding/values-staging.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    repository: docker.io/italia/developers-italia-onboarding
+    tag: latest
+    pullPolicy: Always
+
+  deployEnvironment: "staging"
+
+  env:
+    onboarding_port: "3000"
+    onboarding_environment: "staging"
+    onboarding_app_base_url: "https://onboarding-staging.zland.org"
```
Comment (bfabio) [inline onboarding/values-staging.yaml:19]: Temporary domain

File: onboarding/values-staging.yaml (hunk 1 above)
Comment (sebbalex) [reply onboarding/values-staging.yaml:26]: I would use `contatti@developers.italia.it`

**Comment by bfabio** [comment]
@sebbalex ready for review

**Review by sebbalex — APPROVED** [review]
:rocket:

**Pull request merged**